src/core_plugins/timelion/server/series_functions/fit.js

```diff
@@ -20,7 +20,16 @@ export default new Chainable('fit', {
   fn: function absFn(args) {
     return alter(args, function (eachSeries, mode) {
 
-      const noNulls = _.filter(eachSeries.data, 1);
+      const noNulls = eachSeries.data.filter((item) => {
+        if (item[1] === 0 || item[1]) {
```

Comment:
consider trimming this down to 

`const noNulls = eachSeries.data.filter((item) => (item[1] === 0 || item[1]))`;